zero_shot_sampling/msa_vae.py

```diff
@@ -86,7 +86,7 @@ def save(
   """Saves a VAE model and its metadata to disk."""
   utils.make_directory(model_path)
 
-  model.save_weights(path.join(model_path, 'weights'))
+  model.save_weights(path.join(model_path, 'model.weights.h5'))
```

Comment:
iirc the advantage of saving in the default tf format instead of h5 is that the model is self-describing; we don't need to build it before restoring the weights. Did things not work with the default file format?

Reply:
Based on [1], this is due to using keras v3 vs. v2. It's nice to have things work with v3, so let's stick to your approach rather than pinning things to an older version of keras.

[1] https://discuss.ai.google.dev/t/how-to-properly-save-and-load-model-weights/23916